src/main/java/com/doongji/nestalk/controller/v1/friend/FriendController.java

```diff
@@ -37,4 +35,19 @@
         ));
     }
 
+    @ApiOperation(value = "친구 이름 변경 (JWT 필요)")
+    @PatchMapping("friend")
+    public ResponseEntity<FriendDto> nickNameUpdate(
```

Comment:
```suggestion
    public ResponseEntity<FriendDto> nicknameUpdate(
```
nickname은 한 단어이니 변경해주면 좋을 것 같아요

Reply:
전체적으로 변경하면서 이 부분을 놓쳤나보네요. 감사합니다 :)